sentry_sdk/integrations/django/asgi.py

```diff
@@ -16,24 +18,18 @@
 from sentry_sdk.integrations.asgi import SentryAsgiMiddleware
 from sentry_sdk.utils import capture_internal_exceptions
 
-from django.core.handlers.wsgi import WSGIRequest
-
 
 if TYPE_CHECKING:
-    from typing import Any
-    from typing import Dict
-    from typing import Union
-    from typing import Callable
+    from typing import Any, Callable, Dict, Union
```

Comment:
[`typing.Callable`](https://docs.python.org/3/library/typing.html#typing.Callable) and [`typing.Dict`](https://docs.python.org/3/library/typing.html#typing.Dict) are both deprecated since Python 3.9. We should use `collections.abc.Callable` and the builtin `dict` instead, respectively. (Unless we need these for a compatibility reason -- however, I assume that locally when using type checking, we will typically be able to use newer Python versions, anyways.)

Reply:
Thanks, changed.